### PR TITLE
Refresh test machine image for Fedora 22

### PR DESCRIPTION
**Pull request by mvollmer**
Depends on #2429.

### CODE REVIEW

**Comment by mvollmer** [comment]
The new image has been created and published as expected.  (Everyone should now be able to check this branch out and run the tests locally.)

Some tests are failing and I'll check those.

**Comment by mvollmer** [comment]
There is trouble with testRaid still.  It fails quite often like so:

```
======================================================================
ERROR: testRaid (__main__.TestStorage)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./check-storage", line 440, in testRaid
    b.wait_in_text("#raid_detail_state", "Not running")
  File "/home/mvo/work/cockpit/test/testlib.py", line 247, in wait_in_text
    return self.wait_js_func('ph_in_text', selector, text)
  File "/home/mvo/work/cockpit/test/testlib.py", line 223, in wait_js_func
    return self.phantom.wait("%s(%s)" % (func, ','.join(map(jsquote, args))), timeout=self.phantom_wait_timeout)
  File "/home/mvo/work/cockpit/test/testlib.py", line 629, in wait
    return self.run({'cmd': 'wait', 'cond': cond, 'timeout': timeout*1000})
  File "/home/mvo/work/cockpit/test/testlib.py", line 592, in run
    raise Error("timeout")
Error: timeout

----------------------------------------------------------------------
Ran 1 test in 87.681s

FAILED (errors=1)
```

**Comment by mvollmer** [comment]
> There is trouble with testRaid still. 

Managed to fix this, but now udiskd is crashing frequently: https://bugs.freedesktop.org/show_bug.cgi?id=91011

Not sure what we can do about that, other than waiting for the fix.

**Comment by mvollmer** [comment]
mdraid support in udisks2 is a bit shaky... It seems to get its internal data structures in a mess where it sometimes misses the disappearance of the raid device because some UUID confusion.

**Comment by mvollmer** [comment]
The NetworkManager breakage happened with a version that was pushed to updates-testing "just to see what happens", IIUC.  With that in mind, let's disable updates-testing for our "fedora-22" OS config, and later make a "fedora-22-testing" config that we use information only, but not to gate the pull requests.

**Comment by stefwalter** [comment]
> The NetworkManager breakage happened with a version that was pushed to updates-testing "just to see what happens", IIUC. With that in mind, let's disable updates-testing for our "fedora-22" OS config, and later make a "fedora-22-testing" config that we use information only, but not to gate the pull requests.

Did you file and upstream bug?

**Comment by mvollmer** [comment]
> Did you file and upstream bug?

It was already known: https://bugzilla.redhat.com/show_bug.cgi?id=1234121

File: pkg/shell/cockpit-storage.js [hunk 1]
```diff
@@ -1845,44 +1845,39 @@ PageStorageDetail.prototype = {
 
         is_running = !!block;
 
+        state = is_running? _("Running") : _("Not running");
```
Comment (stefwalter) [inline pkg/shell/cockpit-storage.js]: It seems like this logic is duplicated with the if statement below.

File: pkg/shell/cockpit-storage.js [hunk 2]
```diff
@@ -1845,44 +1845,39 @@ PageStorageDetail.prototype = {
...
+            state = _("Not running");
+        }
+
+        action = { "idle" : "",
+                   "check" : _("Data Scrubbing"),
+                   "repair" : _("Data Scrubbing and Repair"),
+                   "resync" : _("Resyncing"),
+                   "recover" : _("Recovering "),
+                   "frozen" : _("Frozen")
+                 }[raid.SyncAction] || raid.SyncAction;
```
Comment (stefwalter) [inline pkg/shell/cockpit-storage.js]: I would suggest building this map once, and not every time this function is called.

File: pkg/shell/cockpit-storage.js [hunk 3]
```diff
@@ -1845,44 +1845,39 @@ PageStorageDetail.prototype = {
...
+        if (action && action != "idle") {
+            percent = Math.round(raid.SyncCompleted * 100).toString();
+            if (raid.SyncRate > 0) {
+                action_state = cockpit.format(_("$action, ${percent}% complete at $rate"),
+                                              { action: action, percent: percent,
+                                                rate: fmt_size(raid.SyncRate) + "/s" });
```
Comment (stefwalter) [inline pkg/shell/cockpit-storage.js]: Should we use ```cockpit.format_bytes_per_sec()```?

File: pkg/shell/cockpit-storage.js (hunk 1 above)
Comment (mvollmer) [inline pkg/shell/cockpit-storage.js]: Good catch.  This line can be removed.

File: pkg/shell/cockpit-storage.js (hunk 2 above)
Comment (mvollmer) [inline pkg/shell/cockpit-storage.js]: Is it enough to do this for the new version of the code in #2308?

File: pkg/shell/cockpit-storage.js (hunk 3 above)
Comment (mvollmer) [inline pkg/shell/cockpit-storage.js]: In #2308?

File: pkg/shell/cockpit-storage.js (hunk 2 above)
Comment (stefwalter) [inline pkg/shell/cockpit-storage.js]: Ah, sure.

**Pull request closed without merging**